src/magicproxy/proxy.py

```diff
@@ -105,12 +110,36 @@ def proxy_api(path):
             401,
         )
 
+    path = _clean_path_queries(path)
+
     return _proxy_request(
         request=flask.request,
         url=f"{GITHUB_API_ROOT}/{path}",
         headers={"Authorization": f"Bearer {token_info.github_token}"},
     )
 
+def _clean_path_queries(path) -> str:
```

Comment:
Instead of duplicating these, put them in a separate, shared module.

Reply:
Will do :+1: